src/components/ContentNode.vue

```diff
@@ -332,6 +334,14 @@ function renderNode(createElement, references) {
         scopedSlots,
       });
     }
+    case BlockType.links: {
```

Comment:
Is there a DocC PR for this somewhere?

This is more of a comment on the Render Node API than these code changes here, but it's starting to be concerning how this `ContentNode` component is beginning to be conflated with types of layout sections. In my opinion, this kind of directive is better expressed as a new type of primary content section rather than a new type of content itself, similar to how the existing Topics section is expressed. The surface area of this component has expanded very rapidly in the last week or two!

Reply:
No PR yet, this is just based on some draft specs I got.

Reply:
We should make a note to test this as soon as one is ready in case there are any changes needed to make.